app/components/page-components/Chatbot/windows/ChatbotCustomCommandWindow.vue.ts

```diff
@@ -0,0 +1,154 @@
+import { Component } from 'vue-property-decorator';
+import ChatbotWindowsBase from 'components/page-components/Chatbot/windows/ChatbotWindowsBase.vue';
+import ChatbotAliases from 'components/page-components/Chatbot/shared/ChatbotAliases.vue';
+import { ITime } from 'components/shared/inputs/TimePickerInput.vue';
+import { cloneDeep } from 'lodash';
+import { ITab } from 'components/Tabs.vue';
+
+import {
+  ICustomCommand,
+} from 'services/chatbot/chatbot-interfaces';
+
+import {
+  IListMetadata,
+  ITextMetadata,
+  ITimeMetadata,
+} from 'components/shared/inputs/index';
+
+@Component({
+  components: {
+    ChatbotAliases,
+  }
+})
+export default class ChatbotCustomCommandWindow extends ChatbotWindowsBase {
+  newCommand: ICustomCommand = {
+    command: null,
+    response: null,
+    response_type: 'Chat',
+    permission: {
+      level: 163,
```

Comment:
Is this a number that could be constructed with `EChatbotPermissions` to make it more obvious where this magic number is coming from?